lib/src/components/NxBinaryDonutChart/NxBinaryDonutChart.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { forwardRef } from 'react';
+import { Props, propTypes } from './types';
+export { Props } from './types';
+
+import './NxBinaryDonutChart.scss';
+
+const NxBinaryDonutChart = forwardRef<SVGSVGElement, Props>(
+    function NxBinaryDonutChart(props, ref) {
+      const innerRadiusPercent = props.innerRadiusPercent === 0 ? props.innerRadiusPercent :
+        (props.innerRadiusPercent || 50);
```

Comment:
```suggestion
      const innerRadiusPercent = props.innerRadiusPercent == null ? 50 : props.innerRadiusPercent;
```

Reply:
Changed in https://github.com/sonatype/sonatype-react-shared-components/pull/172/commits/0b2fad02e32cf9927810950d59c52abe24ac9a2f thanks